lib/model.js

```diff
@@ -2589,6 +2589,7 @@ class Model {
    * @param  {Object}       values
    * @param  {Object}       options
    * @param  {Object}       options.where                   Options to describe the scope of the search.
+   * @param  {Boolean}      [options.paranoid=true] If true, only non-deleted records will be updated. If false, both deleted and non-deleted records will be updated. Only applies if `options.paranoid` is true for the model.
```

Comment:
Wasn't sure if you want these comments aligned or not. Some are and some aren't.